components/inputs/input-number.js

```diff
@@ -390,6 +390,9 @@ class InputNumber extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixi
 		});
 
 		if (checkValidity) {
+			const inputTextElem = this.shadowRoot.querySelector('d2l-input-text');
```

Comment:
fwiw, this is exactly what is done further below inside of the `validate()` method.